rivalcfg/devices/rival700.py

```diff
@@ -61,6 +68,28 @@
             "default": 1000,
         },
 
+        "logo_rgbgradient": {
```

Comment:
```suggestion
        "logo_color": {
```

This will replace the other method that supports only static colors, we do not need both :)